src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -121,7 +129,10 @@ public Task SuspendObserver()
             }
 
             RedirectToLeader();
-
+            
+            if (TrafficWatchManager.HasRegisteredClients)
```

Comment:
this should go inside the `if (ServerStore.IsLeader())` clause

Reply:
Done